tests/python/pants_test/backend/graph_info/tasks/test_filter.py

```diff
@@ -304,4 +306,55 @@ def test_filter_tag_regex(self):
     self.assert_console_raises(TaskError,
       targets=self.targets('::'),
       options={'tag_regex': ['abc)']}
+                               )
+
+
+class FilterTestForTargetAlias(BaseFilterTest):
+
+  def setUp(self):
+    super(FilterTestForTargetAlias, self).setUp()
```

Comment:
Should use the simper Py3 super.
```suggestion
    super().setUp()
```